src/core_landice/mode_forward/Interface_velocity_solver.cpp

```diff
@@ -113,11 +114,13 @@ void velocity_solver_set_parameters(double const* gravity_F, double const* ice_d
                          double const* sea_level_F, double const* flowParamA_F,
                          double const* flowLawExponent_F, double const* dynamic_thickness_F,
                          double const* clausius_clapeyron_coeff,
+                         double const* thermal_thickness_limit_F,
```

Comment:
Isn't there a mask for the thickness? If so you could replace `li_mask_ValueIce`  with that mask. If not this is fine, but should we get rid of `li_mask_valueIce`?

Reply:
In MPAS we don't have a mask value for the thermal thickness limit and it can differ from the dynamic thickness limit.  Doing it this way is kind of inconsistent with the way we otherwise use mask values, but this was easier than adding the thermal thickness limit to the mask field.